public/app/features/canvas/elements/button.tsx

```diff
@@ -40,20 +43,30 @@ export const defaultStyleConfig: ButtonStyleConfig = {
 class ButtonDisplay extends PureComponent<CanvasElementProps<ButtonConfig, ButtonData>> {
   render() {
     const { data } = this.props;
+    const styles = getStyles(config.theme2, data);
+
     const onClick = () => {
       if (data?.api && data?.api?.endpoint) {
         callApi(data.api);
       }
     };
 
     return (
-      <Button type="submit" variant={data?.style?.variant} onClick={onClick}>
+      <Button type="submit" variant={data?.style?.variant} onClick={onClick} className={styles.button}>
         {data?.text}
       </Button>
     );
   }
 }
 
+const getStyles = stylesFactory((theme: GrafanaTheme2, data: ButtonData | undefined) => ({
+  button: css`
+    position: absolute;
+    height: 100%;
+    width: 100%;
+  `,
+}));
```

Comment:
Build is failing due to a lint issue where we expect styles to written as objects 😅 after fixing this the build should go green 🤞 
```suggestion
const getStyles = stylesFactory((theme: GrafanaTheme2, data: ButtonData | undefined) => ({
  button: css({
    position: 'absolute',
    height: '100%',
    width: '100%',
  }),
}));
```